prow/cmd/sinker/main.go

```diff
@@ -151,6 +159,44 @@ type sinkerReconciliationMetrics struct {
 	podRemovalErrors map[string]int
 }
 
+// Prometheus Metrics
+var (
+	sinkerMetrics = struct {
+		podsCreated      prometheus.Gauge
+		timeUsed         prometheus.Gauge
+		podsRemoved      *prometheus.GaugeVec
+		podRemovalErrors *prometheus.GaugeVec
+	}{
+		podsCreated: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "pods_created",
```

Comment:
Let's try to follow the [naming conventions](https://prometheus.io/docs/practices/naming/) -- here it would be very useful to prefix with `sinker_`

Reply:
This is also a bit misleading -- these are not created by sinker but rather this is a measure of what exists

Reply:
Done.